Sources/TuistAutomation/XcodeBuild/XcodeBuildController.swift

```diff
@@ -213,6 +213,11 @@ public final class XcodeBuildController: XcodeBuildControlling {
             .asSingle()
     }
 
+    public func version() -> Observable<SystemEvent<String>> {
+        let command = ["/usr/bin/xcrun", "xcodebuild", "-version"]
+        return System.shared.observable(command).mapToString()
+    }
+
```

Comment:
Depending on how we feel about it, this also could just be the Swift version since in theory Xcode versions that use the same version of Swift could share a cache, so `xcrun swift --version | head -1` or something would work.